man/systemd.network.xml

```diff
@@ -1237,6 +1237,17 @@
             <para>Allow setting custom port for the DHCP client to listen on.</para>
           </listitem>
         </varlistentry>
+
+        <varlistentry>
+          <term><varname>PromoteSecondaries=</varname></term>
+          <listitem>
+            <para>When true (the default), systemd-networkd will automatically
+            set net.ipv4.conf.$ifname.promote_secondaries to 1 if this
+            sysctl is unset.
```

Comment:
please explain the actual effect of this option here, rather than how it is implemented. When reading this, I a mstill puzzled about what this actually does

Reply:
Thanks, good point. I fixed this now and added an explanation. Feedback welcome (as I'm not a native speaker), happy to rework for more clarity if needed.